File: tests/js-kit/rps/rock.js
const Agent = require('./agent');
const agent = new Agent();
agent.initialize().then(async () => {
  while(true) {
    console.log('R'); // tell the match you want to play Rock in the game
    agent.endTurn(); // end turn
    await agent.update(); // wait for updates
  }
});

File: tests/c-kit/rps/rock.c

#include "kit.h"
#include <stdio.h>

int main() {
  // initialize
  agent_initialize();
  // we have agent of type Agent * available now
  fprintf(stdout, "R\n");
  while(1) {
    // play rock
    fprintf(stdout, "R\n");
    // end turn
    agent_end_turn();
    fflush(stdout);
    // wait for updates
    agent_update();
  }
  
  return 0;
}